ios/BT/API/Model/CheckInStatus.swift

```diff
@@ -0,0 +1,39 @@
+import ExposureNotification
+import Foundation
+import RealmSwift
+
+@objcMembers
+class CheckInStatus: Object, Codable {
+  @objc dynamic var id: String = .default
+  @objc dynamic var date: Int = 0
+  var feelingGood: Bool = false
```

Comment:
At the risk of jumping the gun here. Is there a world where this becomes more than a flag? Maybe the `feeling` can hold 3 states? Should we store an enum/integer here? cc @juancruzgs we do have an enum already on the typescript side. Definitely an improvement to make down the road but if we are just starting maybe a 5 min chat is worth it?

Reply:
If there are more than 2 states to store then yes, we can change this to a Integer